modules/util/portal-tools-rest-builder-test-impl/rest-openapi.yaml

```diff
@@ -33,123 +58,130 @@ components:
                     type: string
                 description:
                     type: string
-                documentsRepository:
-                    $ref: "#/components/schemas/Folder"
-                externalField:
-                    items:
-                        $ref: "src/test/resources/com/liferay/portal/tools/rest/builder/external-dependencies/rest-openapi.yaml#ExternalReferenceElement1"
-                    type: array
                 id:
                     format: int64
                     type: integer
                 name:
                     type: string
-                self:
-                    type: string
-                subFolders:
-                    $ref: "#/components/schemas/Folder"
+                testObject:
+                    $ref: "#/components/schemas/TestObject"
             type: object
-        Test:
+        TestObject:
             description:
-                Test Component to test the REST Builder support for advanced features
+                "https://www.schema.org/Document"
+            discriminator:
+                mapping:
```

Comment:
Should we include a validation in the REST Builder to check that `x-merge-properties` is always missing (or true) when the Schema referred here contains an allOf?